subprojects/execution/src/test/groovy/org/gradle/internal/execution/steps/ExecuteStepTest.groovy

```diff
@@ -51,13 +51,14 @@ class ExecuteStepTest extends StepSpec<InputChangesContext> {
 
         then:
         result.executionResult.get().outcome == expectedOutcome
-        result.duration.toMillis() >= 3
+        // Check
+        result.duration.toMillis() >= 100
 
         _ * context.inputChanges >> Optional.empty()
         _ * work.execute({ UnitOfWork.ExecutionRequest executionRequest ->
             executionRequest.workspace == workspace && !executionRequest.inputChanges.present && executionRequest.previouslyProducedOutputs.get() == previousOutputs
         }) >> {
-            sleep 3
+            sleep 200
```

Comment:
Can we maybe do something like 20ms and 10ms instead of 200ms and 100ms? I think 200ms is quite a lot for a unit test.